mpu6050_scripts/mpu6050.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#check WHO_AM_I register
+WAMI=$(sudo i2cget -y 0 0x68 0x75)
+if [ "$WAMI" != "0x68" ]
+then
+  echo "MPU6050 not found ($WAMI)"
+  exit 1
+fi
+
+#start conversion: set PWR_MGMT_1 to 0
+PM1=$(sudo i2cget -y 0 0x68 0x6B)
+if [ "$PM1" != "0x00" ]
+then
+  sudo i2cset -y 0 0x68 0x6B 0
+  echo "MPU6050: start conversion..."
+  sleep 1
+fi
+
+while true
+do
+ echo -n "MPU6050 T: "
+ TH=$(sudo i2cget -y 0 0x68 0x41)
```

Comment:
It is better to get 2 bytes in one command using -w option.

Reply:
I know about -w option, but I can't remember for now why I used byte-wide reading. Maybe -w didn't work, maybe because I had some experiments with DS1621 i2c temperature sensor chip before and the main shell script was left unchanged for MPU6050. I can't check it for now without board and MPU6050.

Reply:
The mean thing - it works...